tslint.json

```diff
@@ -1,11 +1,48 @@
 {
-  "extends": "@salesforce/dev-config/tslint",
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended"],
+  "jsRules": {},
   "rules": {
+    "arrow-parens": [true, "ban-single-arg-parens"],
+    "curly": [true, "ignore-same-line"],
+    "interface-name": false,
+    "interface-over-type-literal": false,
+    "max-classes-per-file": false,
+    "max-line-length": false,
+    //"no-any": true,
+    "no-bitwise": false,
+    "no-console": false,
+    "no-empty": false,
+    "no-floating-promises": true,
+    "no-namespace": false,
+    "no-string-literal": false,
+    "no-trailing-whitespace": [true],
+    "object-literal-key-quotes": [true, "as-needed"],
+    "object-literal-sort-keys": false,
+    "only-arrow-functions": [true, "allow-named-functions"],
+    "ordered-imports": true,
+    //"quotemark": [true, "single", "avoid-template", "avoid-escape"],
+    "semicolon": [true, "always", "ignore-bound-class-methods"],
+    "trailing-comma": [true, { "multiline": "never", "singleline": "never" }],
+    "variable-name": [true, "allow-leading-underscore"],
+    // -- Begin MS SDL https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle
+    // "no-banned-terms": true,
+    // "no-delete-expression": true,
+    // "no-document-domain": true,
+    // "no-disable-auto-sanitization": true,
+    // "no-duplicate-parameter-names": true,
+    // "no-exec-script": true,
+    // "no-function-constructor-with-string-args": true,
+    // "no-octal-literal": true,
+    // "no-string-based-set-immediate": true,
+    // "no-string-based-set-interval": true,
+    //"no-string-based-set-timeout": true,
+    "no-eval": true,
+    // -- End MS SDL
     "no-reserved-keywords": false,
     "no-any": false,
     "quotemark": [true, "single", "avoid-escape"],
     "member-ordering": false,
-    "no-string-based-set-timeout": false,
-    "semicolon": [true, "always", "ignore-bound-class-methods"]
+    "no-string-based-set-timeout": false
```

Comment:
said it previously, but had to copy this over from the old version of the salesforce utils package that needed to upgrade with typescript.